composer.json

```diff
@@ -31,12 +31,12 @@
 		"php": ">=5.6.0"
 	},
 	"require-dev": {
-		"phpcompatibility/phpcompatibility-wp": "^2.1.0",
+		"phpcompatibility/phpcompatibility-wp": "^2.1.3",
 		"dealerdirect/phpcodesniffer-composer-installer": "^0.7.1",
-		"squizlabs/php_codesniffer" : "^3.5.4",
-		"wp-coding-standards/wpcs": "*",
+		"squizlabs/php_codesniffer" : "^3.6.0",
+		"wp-coding-standards/wpcs": "~2.3.0",
 		"wp-phpunit/wp-phpunit": "^5.9",
-		"phpunit/phpunit": "^7.5",
+		"phpunit/phpunit": "^9.0",
```

Comment:
I'd recommend removing this line altogether. Let's leave `yoast/phpunit-polyfills` to decide which version to install based on the current PHP version.

`composer.lock` should also be removed from source control so that we don't lock packages per PHP version.

Reply:
Ok, I've removed the line 👍

Reply:
I've also removed composer.lock